src/main/java/org/skriptlang/skriptworldguard/elements/expressions/ExprRegionFlag.java

```diff
@@ -0,0 +1,129 @@
+package org.skriptlang.skriptworldguard.elements.expressions;
+
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import com.sk89q.worldedit.bukkit.BukkitAdapter;
+import com.sk89q.worldguard.WorldGuard;
+import com.sk89q.worldguard.protection.flags.*;
+import com.sk89q.worldguard.protection.managers.RegionManager;
+import com.sk89q.worldguard.protection.regions.ProtectedRegion;
+import com.sk89q.worldguard.protection.regions.RegionContainer;
+import org.bukkit.World;
+import org.bukkit.event.Event;
+import org.skriptlang.skriptworldguard.SkriptWorldGuard;
+
+public class ExprRegionFlag extends SimpleExpression<String> {
+
+    static {
+
+        Skript.registerExpression(ExprRegionFlag.class, String.class, ExpressionType.COMBINED, "[the] flag %string% of region %string% in [world] %world%");
+
+    }
+
+    private Expression<String> flag;
+    private Expression<String> region;
+    private Expression<World> world;
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public boolean init(Expression<?>[] expression, int arg1, Kleenean arg2, SkriptParser.ParseResult arg3) {
+        flag = (Expression<String>) expression[0];
+        region = (Expression<String>) expression[1];
+        world = (Expression<World>) expression[2];
+        return true;
+
+    }
+
+    @Override
+    protected String[] get(Event event) {
+        Flag<?> fl = null;
+
+        WorldGuard wg = WorldGuard.getInstance();
+        RegionContainer container = wg.getPlatform().getRegionContainer();
+        RegionManager regions = container.get(BukkitAdapter.adapt(world.getSingle(event)));
+        ProtectedRegion rg = regions.getRegion(region.getSingle(event));
+
+        fl = Flags.fuzzyMatchFlag(wg.getFlagRegistry(), flag.getSingle(event));
+
+        Object value = rg.getFlag(fl);
+
+        return new String[]{value.toString()};
+    }
+
+    @Override
+    public String toString(Event arg0, boolean arg1) {
+        return "WorldGuard region flag";

```

Comment:
the toString should represent the actual input of the user
ex: `flag <flag> of <region>` (take a look at some of the other syntax elements too :))